webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersAdministration.m.css

```diff
@@ -27,3 +27,11 @@
 .tabPanel {
   overflow: auto;
 }
+
+.tabInner {
+  height: 100%;
+}
+
+.tabOuter {
+  height: 100%;
+}
```

Comment:
You must not mix styles you will pass to `SContext` with component styles.
For example, if your target is `Tab.m.css`, then you can make a new style file `UsersAdministrationTab.m.css`
In general, all styles in the file must be used with `s` and passed to the component's children as `className`, or the whole file will be passed to `SContext` (in that case, you are allowed to use some anchor class from this style module to add more specificity to styles)

Reply:
done for all similar cases